pro/src/ui-kit/typography/Typography.tsx

```diff
@@ -0,0 +1,41 @@
+import cn from 'classnames'
+import React from 'react'
+
+import styles from './Typography.module.scss'
+
+interface ITypographyProps {
+    level: 'h1' | 'h2' | 'h3' | 'h4';
+    title: string;
+    className?: string;
+}
+
+const Typography = ({ level, title, className }: ITypographyProps): JSX.Element => {
+  switch(level) {
+    case 'h1':
+      return (
+        <h1 className={cn(styles.h1, className)}>
+          {title}
+        </h1>
+      )
+    case 'h2':
+      return (
+        <h2 className={cn(styles.h2, className)}>
+          {title}
+        </h2>
+      )
+    case 'h3':
+      return (
+        <h3 className={cn(styles.h3, className)}>
+          {title}
+        </h3>
+      )
+    case 'h4':
+      return (
+        <h4 className={cn(styles.h4, className)}>
+          {title}
+        </h4>
+      )
+  }
+}
```

Comment:
Proposition : 

1. utiliser children pour plus de flexibilité
2. ajouter des tags possible
3. associer n'importe quel style avec n'importe quel tag

```suggestion
interface ITitleProps {
  level: 1 | 2 | 3 | 4;
  as?: 'p' | 'span' | 'h1' | 'h2' | 'h3' | 'h4';
  className?: string;
  children: JSX.Element, JSX.Element[]; // de mémoire
}

const Title = ({ level, as, className, children }: ITitleProps): JSX.Element => {
  const CustomTag = as || `h${level}`
  
  return (
    <CustomTag className={cx(styles[`title-${level}`], className)}>
      { children }
    </CustomTag>
  )
}
```